README.md

```diff
@@ -117,6 +117,58 @@ ollama run mario
 Hello! It's your friend Mario.
 ```
 
+### Edit existing modelfile to reduce VRAM usage
+
+1. Get the model file for an existing model 
+
+Assuming that your model name is `nous-hermes:13b` -- replace this with your model name
+
+
+```bash
+ollama show --modelfile nous-hermes:13b > custom.modelfile
+```
+
+2. Add the settings you want to change to the modelfile using your favorite editor
+
+For example, to attempt to load 20 layers into the GPU, and leave the rest of them to the CPU; and to use only 4 CPU threads, add the following lines to the modelfile:
+
+```yaml
+PARAMETER num_gpu 20
+PARAMETER num_thread 4
+```
+
+Note that if you use a non-dedicated GPU (e.g. a laptop GPU), you may need to leave some free VRAM for other applications to use, 
```

Comment:
This section could work great in the https://github.com/jmorganca/ollama/blob/main/docs/faq.md as Ollama should, over time, get better at scheduling these layers automatically.